engine/src/main/java/org/terasology/engine/world/chunks/pipeline/ChunkProcessingPipeline.java

```diff
@@ -44,120 +32,162 @@
  */
 public class ChunkProcessingPipeline {
```

Comment:
I think I would like to just do away with this ChunkProcessingPipeline. that could be addressed later